fb-negotiation/core/event.py

```diff
@@ -5,3 +5,7 @@ class Event(BaseEvent):
     def SelectEvent(agent, data, time=None):
         return Event(agent, time, 'select', data)
 
+    @staticmethod
+    def RejectEvent(agent, time=None):
+        return Event(agent, time, 'reject', None)
+
```

Comment:
I thought we agreed to remove this event?

Reply:
Yes, I only had it to help with debugging.  The sessions don't actually use it (when the system is running correctly) and accordingly, I've commented it out.